domains/ray.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "RayhanADev's Website!",
+  "repo": "https://github.com/rayhanadev/site",
+  "owner": {
+    "username": "rayhanadev",
+    "email": "rayhan.arayilakath@wa-students.org"
+  },
+  "record": {
+    "CNAME": "its.rayhanadev.repl.co"
```

Can you confirm if `repl.co` allows CNAME records pointing to it? If not, I'd suggest either going with [github pages](https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/creating-a-github-pages-site) or you can also use a URL redirection to your repl.

So either,

```suggestion
    "CNAME": "rayhanadev.github.io"
```

**OR**

```suggestion
    "URL": "https://its.rayhanadev.repl.co"
```

---

Done!